types/value.go

```diff
@@ -4,8 +4,11 @@ import (
 	"fmt"
 )
 
+var ErrDatetime = fmt.Errorf("error parsing datetime value")
```

Comment:
I'm not sure if these need to be public, maybe they could be dropped in an internal package.